mongodbatlas/organizations.go

```diff
@@ -62,6 +68,24 @@ type Organizations struct {
 	TotalCount int             `json:"totalCount"`
 }
 
+// InvitationOptions filtering options for invitations.
+type InvitationOptions struct {
+	Username string `url:"username,omitempty"`
+}
+
+// Invitation represents the structure of an Invitation.
+type Invitation struct {
+	ID              string   `json:"id,omitempty"`
+	OrgID           string   `json:"orgId,omitempty"`
+	OrgName         string   `json:"orgName,omitempty"`
+	CreatedAt       string   `json:"createdAt,omitempty"`
+	ExpiresAt       string   `json:"expiresAt,omitempty"`
+	InviterUserName string   `json:"inviterUserName,omitempty"`
```

Comment:
[q] can you confirm this is correct, I have a feeling it should be `inviterUsername`

Reply:
yes, it is `inviterUsername`